Formula/conduit.rb

```diff
@@ -7,13 +7,22 @@ class Conduit < Formula
   head "https://github.com/ConduitIO/conduit.git", branch: "main"
 
   depends_on "go"
+  depends_on "node"
+  depends_on "yarn"
 
   def install
-    system "make", "build-server"
+    system "make"
     bin.install "conduit"
   end
 
   test do
-    shell_output("#{bin}/conduit -version").match(/0.5.0/)
+    # Assert conduit version
+    assert_match(version.to_s, shell_output("#{bin}/conduit -version"))
+    # Run conduit
+    log = shell_output("#{bin}/conduit")
+    # Check that gRPC server is running on port 8084
+    assert_match("grpc server started address=[::]:8084", log)
+    # Check that HTTP server is running on port 8080
+    assert_match("http server started address=[::]:8080", log)
   end
```

Comment:
Can we define what ports it should use? These might be in use.

Reply:
yes we have a the flags `grpc.address` and `http.address` to define them

Reply:
@SMillerDev I created a test that runs conduit and asserts that the servers are running, but to get the output I have to run conduit and stop it, because it doesn't stop by itself, and for the I used the library "Open3", but it seems like this library doesn't load for some machines in the tests, any suggestions of what I should use instead?